angular/demo/daisyui/src/app/samples/toast/default.route.ts

```diff
@@ -0,0 +1,20 @@
+import {Component} from '@angular/core';
+import {ToastComponent} from './toast.component';
+
+@Component({
+	standalone: true,
+	imports: [ToastComponent],
+	template: `<div class="h-[215px]">
+		<button class="btn btn-primary" (click)="reset()">Reset Toast</button>
+		<div class="toast">
+			<app-toast [className]="'alert-success'" [(visible)]="visible">This is a success toast</app-toast>
```

Comment:
you can remove the [] from className and directly pass the alert-success without ''